app/controllers/user_challenges_controller.rb

```diff
@@ -27,6 +28,28 @@ def destroy
     end
   end
 
+  def evaluate_comments
+    evaluator = UserChallenge::Evaluator.new(params: params)
+    result = evaluator.save_comments
+
+    if result.success?
+      head :no_content
+    else
+      render json: { resource: 'user_challenge', errors: [user_challenge: result.errors.first] }, status: 500
```

Comment:
Yea a 422 is usually what I'll do in my projects, but the rest of the actions were 500s, so I was following it for consistency.

@atareshawty if you want I can run through change status codes!